ibazel.hcl

```diff
@@ -0,0 +1,29 @@
+description = "Tools for building Bazel targets when source files change."
+binaries = ["ibazel"]
+source = "https://github.com/bazelbuild/bazel-watcher/releases/download/v${version}/ibazel_${os}_${arch_}"
+
+vars = {
+  "arch_": "${arch}",
+}
+
+on "unpack" {
+  rename {
+    from = "${root}/ibazel_${os}_${arch_}"
+    to = "${root}/ibazel"
+  }
+}
+
+platform "darwin" {
+  vars = {
+    "arch_": "arm64",
+  }
+}
```

Comment:
This prevents an error when generating the digests below (as `darwin_arm64` is not available in the github releases for the `0.26.2` version), but I'm not sure if it's actually a good approach?